### PR TITLE
Add webpack watch gulp task

### PR DESCRIPTION
**Pull request by aeisenberg**
Now, when running `npm run watch`, both the regular tsc command
and the webpack command will be run in watch mode.

The raw gulp tasks are now:

- `gulp watchView` to watch webpack compilation.
- `gulp watchCss` to watch for css changes.
- `gulp compileView` to compile the webpack once and exit.

However, stats are no longer being printed out. Not sure why.

## Checklist

- [n/a] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [x] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

**Comment by aeisenberg** [comment]
Fixes https://github.com/github/vscode-codeql/issues/1245

File: extensions/ql-vscode/gulpfile.ts/webpack.ts [hunk 1]
```diff
@@ -2,7 +2,24 @@ import * as webpack from 'webpack';
...
 
 export function compileView(cb: (err?: Error) => void) {
-  webpack(config).run((error, stats) => {
+  doWebpack(config, true, cb);
+}
+
+export function watchView(cb: (err?: Error) => void) {
+  const watchConfig = {
+    ...
+    config,
```
Comment (charisk) [inline extensions/ql-vscode/gulpfile.ts/webpack.ts:11]: Minor, but can we change that to `...config`?

**Review by charisk — APPROVED** [review]
This is awesome! Thanks for looking into it. Just one minor comment.

File: extensions/ql-vscode/gulpfile.ts/webpack.ts (hunk 1 above)
Comment (aeisenberg) [inline extensions/ql-vscode/gulpfile.ts/webpack.ts:11]:
```suggestion
    ...config,
```

**Pull request merged**